templates/components/molecules/icon-block/icon-block.html.twig

```diff
@@ -0,0 +1,10 @@
+<div class="decanter-icon-block {{ classes }}">
+  <img class="decanter-icon-block__img" src="../../../img/circle.png" alt="Please provide alt text">
```

Comment:
Please update the src attribute to include a template variable so we can add our own src files when we use this template elsewhere. 

Please also do the same with the alt text.